code/modules/hydroponics/gene_modder.dm

```diff
@@ -130,7 +134,7 @@
 	else if(istype(I, /obj/item/unsorted_seeds))
 		to_chat(user, "<span class='warning'>You need to sort [I] first!</span>")
 		return ..()
-	else if(istype(I, /obj/item/disk/plantgene))
+	else if(istype(I, /obj/item/disk/plantgene) || I.type == /obj/item/storage/box || type2parent(I.type) == /obj/item/storage/box)
```

Comment:
run me through what you're trying to do here? Why not just `istype(I, /obj/item/storage/box)` ???

Reply:
I specifically avoided using istype() throughout the PR but I don't remember why. I'll try substituting it in and see if it causes trouble.

Reply:
wait I did use it here as well. what

Reply:
put it in and it seems to work alright. I have no idea why I avoided using it earlier.